EAD-Session-1-Assignment/functions.js

```diff
@@ -0,0 +1,373 @@
+/**
+ * Class containing methods to validate the form
+ */
+class Validation
+{
+    Validation(){}
+
+    /**
+     * Method to validate the user details
+     * @param {*} name is the user name
+     * @param {*} email is the user email
+     * @param {*} password is the user password
+     * @param {*} confirmPassword is the confirm password field
+     * @param {*} contact is the user contact number
+     * @param {*} gender is the user gender
+     */
+    static validateUserInformation(name , email , password , confirmPassword , contact , gender)
+    {
+        // Validating Contact Number
+        var count=0;
+		var filterContact = new RegExp('([a-zA-Z!@#$%^&*()_+])');
```

Comment:
Change the regex to allow only numeric characters. No need to use filterContact. Use the regex only for numeric allowed characters.

Reply:
Done